src/journeys_utils.js

```diff
@@ -908,3 +908,47 @@ journeys_utils.setJourneyLinkData = function(linkData) {
 	}
 	journeys_utils.journeyLinkData = data;
 };
+
+journeys_utils.getValueForKeyInBranchViewData = function(key) {
+	if(!journeys_utils){
+		return false;
+	}
+
+	if(!journeys_utils.branch){
+		return false;
+	}
+
+	if(!journeys_utils.branch._branchViewData){
+		return false;
+	}
+
+	if(!journeys_utils.branch._branchViewData.data){
+		return false;
+	}
+
+	return journeys_utils.branch._branchViewData.data[key];
+};
+
+journeys_utils.getBranchViewDataItemOrUndefined = function(name){
+	if(journeys_utils.getValueForKeyInBranchViewData(name)){
```

Comment:
hey @jwangler-branch , can you fix the spacing here? There is a linter in the package.json.

Just run `npm install` and it should pick up the hook going forward

Reply:
for example: `if (journeys_utils.getValueForKeyInBranchViewData(name)) {`

similar patterns in the rest of the code

Reply:
Yes, I'll set lint as auto-execute.. :)